src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,246 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
+				'debug' => defined( 'BP_GROUPS_DEFAULT_EXTENSION' ) ? BP_GROUPS_DEFAULT_EXTENSION : 'undefined',
+			),
+			'BP_MEMBERS_SLUG' => array(
+				'label' => 'BP_MEMBERS_SLUG',
+				'value' => defined( 'BP_MEMBERS_SLUG' ) ? BP_MEMBERS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_MEMBERS_SLUG' ) ? BP_MEMBERS_SLUG : 'undefined',
+			),
+			'BP_SIGNUPS_SKIP_USER_CREATION' => array(
+				'label' => 'BP_SIGNUPS_SKIP_USER_CREATION',
+				'value' => defined( 'BP_SIGNUPS_SKIP_USER_CREATION' ) && BP_SIGNUPS_SKIP_USER_CREATION ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+				'debug' => defined( 'BP_SIGNUPS_SKIP_USER_CREATION' ) ? BP_SIGNUPS_SKIP_USER_CREATION : 'undefined',
+			),
+			'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH' => array(
+				'label' => 'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH',
+				'value' => defined( 'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH' ) ? BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH' ) ? BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH : 'undefined',
+			),
+			'BP_NOTIFICATIONS_SLUG' => array(
+				'label' => 'BP_NOTIFICATIONS_SLUG',
+				'value' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : 'undefined',
+			),
+			'BP_BLOGS_SLUG' => array(
+				'label' => 'BP_BLOGS_SLUG',
+				'value' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : 'undefined',
+			),
+			'BP_FRIENDS_SLUG' => array(
+				'label' => 'BP_FRIENDS_SLUG',
+				'value' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : 'undefined',
+			),
+			'BP_FRIENDS_DB_VERSION' => array(
+				'label' => 'BP_FRIENDS_DB_VERSION',
+				'value' => defined( 'BP_FRIENDS_DB_VERSION' ) ? BP_FRIENDS_DB_VERSION : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_FRIENDS_DB_VERSION' ) ? BP_FRIENDS_DB_VERSION : 'undefined',
+			),
+			'BP_EMBED_DISABLE_PRIVATE_MESSAGES' => array(
+				'label' => 'BP_EMBED_DISABLE_PRIVATE_MESSAGES',
+				'value' => defined( 'BP_EMBED_DISABLE_PRIVATE_MESSAGES' ) ? BP_EMBED_DISABLE_PRIVATE_MESSAGES : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_EMBED_DISABLE_PRIVATE_MESSAGES' ) ? BP_EMBED_DISABLE_PRIVATE_MESSAGES : 'undefined',
+			),
+			'BP_EMBED_DISABLE_ACTIVITY_REPLIES' => array(
+				'label' => 'BP_EMBED_DISABLE_ACTIVITY_REPLIES',
+				'value' => defined( 'BP_EMBED_DISABLE_ACTIVITY_REPLIES' ) ? BP_EMBED_DISABLE_ACTIVITY_REPLIES : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_EMBED_DISABLE_ACTIVITY_REPLIES' ) ? BP_EMBED_DISABLE_ACTIVITY_REPLIES : 'undefined',
+			),
+			'BP_USE_WP_ADMIN_BAR' => array(
```

Comment:
I wonder why this is still usable as there's a setting about it.. I'll give it a closer look

Reply:
@renatonascalves after a second look into it, please move it to the bottom (deprecated constants). See: https://buddypress.trac.wordpress.org/ticket/9104

Reply:
Moved! 💪🏾